app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/medication/MedicationDetailsViewModel.kt

```diff
@@ -42,6 +42,7 @@ class MedicationDetailsViewModel @Inject constructor(
                     .getComments(
                         medicationWithSummaryAndPharmacyDto.medicationRecord.prescriptionIdentifier
                     )
+            var userProfileId: String? = null
```

Comment:
userProfileId is hdId, which is available at repo level. Can we remove it from here?